### DISCUSSION TITLE
Headless Chrome shows blank window

### DISCUSSION THREAD

**Discussion by michael-at-mitxp** [lucono/karma-test-explorer#93] [General]
Hi there,

I am using VScode on Windows 11 with latest Chrome. Just recently I noticed a blank white window with no controls at all (so can't be minified) when I work on a project that has the Karma-test-explorer extension enabled. When I switch to non headless mode in the settings I see a chrome instance with the karma debug window at the same place which can be minified.

Anyon else with the same issue? Any solutions?

Thanks a lot,
Michael

**Comment by kheder-87** [comment]
Hello,
I have the same problem since this morning.
I am using Windows 10

**Comment by cristianWilliam** [comment]
Same here

**Reply by cristianWilliam** [reply]
Seems like even when I disable the extension, and run the npm run test with an Angular 16 app, same thing happens.

**Comment by FabioBezerra** [comment]
Hi. Same here

**Comment by Alanoll** [comment]
Looks like it may be something with the version of chrome on the machine.
https://stackoverflow.com/questions/78996364/chrome-129-headless-shows-blank-window

I added 
```
"--window-position=-2400,-2400"
```
To my settings.json for the extensions customLauncher flags to move the window off the screen.
![image](https://github.com/user-attachments/assets/42970cb2-81b3-4cbc-a305-982660b94f50)

**Reply by kheder-87** [reply]
don't work for me

**Comment by stesee** [comment]
Issue went away with Chrome 130.
